File: python/baekjoon/step9/bkjn_5086_1.py
while True:
  num1, num2 = map(int, input().split())
  
  if num1 == 0 and num2 == 0:
    break
  elif num1 % num2 == 0:
    print("multiple")
  elif num2 % num1 == 0:
    print("factor")
  else:
    print("neither")

File: js/baekjoon/step9/bkjn_5086_1.js
const input = require("fs")
  .readFileSync("dev/stdin")
  .toString()
  .trim()
  .split("\n");

for (i = 0; i < input.length - 1; i++) {
  let num = input[i].split(" ").map((item) => item * 1);
  let a = num[0];
  let b = num[1];
  if (b % a === 0) {
    console.log("factor");
  } else if (a % b === 0) {
    console.log("multiple");
  } else {
    console.log("neither");
  }
}
